src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -51,6 +48,8 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: GpPriceS
     fromDecimals: order.inputToken.decimals,
     toDecimals: order.outputToken.decimals,
     validTo: timestamp(order.validTo),
+    userAddress: order.owner,
+    receiver: order.receiver,
```

Comment:
So you think the issue is that, we get different fee because the creation and the checks quotes only change in the later not specifying the `receiver`?

In any case, this change is good, not sure if this solves the issue that intended to solve though, but is good we specify them :)

Reply:
I'm still not 100% sure what was causing the issue in the first place, but looking at the requests when testing with a liquidity provider account, it does make a big difference with/without the user address due to the fee.

With regular accounts is still hard to reproduce, maybe because there isn't much of a difference for them